vnext/ReactUWP/EndPoints/dll/react-native-uwp.x86.def

```diff
@@ -40,9 +40,8 @@ EXPORTS
 ?JsPointerToStringUtf8@react@facebook@@YG?AW4_JsErrorCode@@PBDIPAPAX@Z
 ?JsStringToStdStringUtf8@react@facebook@@YG?AW4_JsErrorCode@@PAXAAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z
 ?LoadJavaScriptInProxyMode@DevSupportManager@uwp@react@@UAE?AV?$function@$$A6G?AV?$unique_ptr@VJSExecutor@react@facebook@@U?$default_delete@VJSExecutor@react@facebook@@@std@@@std@@V?$shared_ptr@VExecutorDelegate@react@facebook@@@2@V?$shared_ptr@VMessageQueueThread@react@facebook@@@2@@Z@std@@ABUDevSettings@3facebook@@@Z
-?LoadJavaScriptInSandboxMode@DevSupportManager@uwp@react@@UAE?AV?$function@$$A6G?AV?$unique_ptr@VJSExecutor@react@facebook@@U?$default_delete@VJSExecutor@react@facebook@@@std@@@std@@V?$shared_ptr@VExecutorDelegate@react@facebook@@@2@V?$shared_ptr@VMessageQueueThread@react@facebook@@@2@@Z@std@@ABUDevSettings@3facebook@@@Z
+;?LoadJavaScriptInSandboxMode@DevSupportManager@uwp@react@@UAE?AV?$function@$$A6G?AV?$unique_ptr@VJSExecutor@react@facebook@@U?$default_delete@VJSExecutor@react@facebook@@@std@@@std@@V?$shared_ptr@VExecutorDelegate@react@facebook@@@2@V?$shared_ptr@VMessageQueueThread@react@facebook@@@2@@Z@std@@ABUDevSettings@3facebook@@@Z
```

Comment:
>;? [](start = 0, length = 2)

Is it a typo? It does not look like a valid symbol name.

Reply:
In case if it is commented out, then it is better just to remove it.

---
In reply to: [370711760](https://github.com/microsoft/react-native-windows/pull/3948#discussion_r370711760) [](ancestors = 370711760)

Reply:
This was a rename of the file that was previously being used for OSS defs. It had it commented out since we compile it out in the non-patched version.

Reply:
I do like how the upstream defs file commented out symbols we couldn't expose due to lacking a patch. This wasn't consistently being used though, and the exports here moved from the previous upstream defs file is definitely a bit out of data.

I'm going to make the non-patched deps follow the patched ones more closely, with them being the same except for the ones we can't yet expose commented out.

Reply:
Why to comment and not to remove?
Having commented out line could still make some developers believe that we export the symbol, and spend time debugging why something doe snot work.

---
In reply to: [371018277](https://github.com/microsoft/react-native-windows/pull/3948#discussion_r371018277) [](ancestors = 371018277)

Reply:
I've gotten use out of them if I wanted to quickly find which of the  symbols we avoid exporting due to lack of patches. This is the approach that was used before for upstream React Native exports, and I thought the intent was relatively clear when I saw it.